src/plugins/flexibleLayout/components/flexibleLayout.vue

```diff
@@ -171,9 +172,11 @@ export default {
         }
     },
     mounted() {
+        this.buildIdentifierMap();
         this.composition = this.openmct.composition.get(this.domainObject);
         this.composition.on('remove', this.removeChildObject);
         this.composition.on('add', this.addFrame);
+        this.composition.load();
 
         this.RemoveAction = new RemoveAction(this.openmct);
```

Comment:
I think you can remove this.RemoveAction and it's import now since you're using composition.remove.

Reply:
Fixed, please review again